broken_linkfinder/README.md

```diff
@@ -0,0 +1,16 @@
+## Broken Link Finder
+
+## Setup and activate virtual environment :
+For Unix based systems please execute the following command to create venv and install requirements.
+```
+make init
+source .venv/bin/activate
+```
+
+## Usage: 
+
```

Comment:
Add more about the purpose of the script and a brief description of how it gonna solve that

Reply:
Updated.